ynput/ayon-core
# Chore: Move unreal client code

## Changelog Description
Moved Unreal client code next to server codebase.

## Additional information
This PR requires update of integration submodule with [this PR](https://github.com/ynput/ayon-unreal-plugin/pull/9) (updated once merged) and probably release of integration plugin to AYON market.

## Testing notes:
1. Create package of Unreal and core.
2. Upload both to server.
3. Make sure AYON launcher is using client code from this PR (e.g using dev paths).
4. Unreal integration should work as before.
